syntax = "proto3";
option java_package = "VireProtobuf.vire.cms";
option java_outer_classname = "InvalidCredentialsErrorStub";
package vire.cms;

import "vire/utility/BaseError.proto";

message InvalidCredentialsError {

  vire.utility.BaseError _base = 1;

  string invalid_login    = 100; // Invalid login
  string invalid_password = 101; // Invalid password

}